### PR TITLE
New content: Fixing website speed cross-functionally

### LINKED ISSUE #1110: content: Fast content - Fixing speed - cross-functionally
**Solving site speed is not only dependent on developers – they need the help from marketing, web designers, the analytics team and stakeholders to succeed. These four articles describe how developers can set up collaboration with each department, and get great help in fixing speed.**
What are you planning to teach folks in this post?

**Target publish date:** `<yyyy-mm-dd>`
- [ ] Check this box if this is a hard deadline.

**Process**
Make a copy of [this template](https://docs.google.com/document/d/1ydauCufwwavStaKxhIDHuNivVxgQBOdDno4uLjGIjmE/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [X] 1. I've done my proposal `<https://docs.google.com/document/d/1L7j3qG4-pbKBu52BmX_p4rcERiIMUzxFX8YNoAxDdug/edit?usp=sharing>`
- [ ] 2. Proposal approved — _reviewer checks this_
- [X] 3. I've added a draft to my proposal doc
- [ ] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request `#<add PR number>`
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by kaycebasques**
Fixes #1110

### CODE REVIEW

File: src/site/content/en/blog/fixing-website-speed-cross-functionally/index.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+   -  **Not used or not owned**. These should be removed.
+-  **Check how much each script slows down your site by comparing site speed with and without it**.
+   In [WebPageTest](https://www.webpagetest.org/) this can be set up
+   via the [Block tab in Advanced Settings](https://andydavies.me/blog/2018/02/19/using-webpagetest-to-measure-the-impact-of-3rd-party-tags/).
+   Or you can [block network requests in Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/network/#block).
+
+### Meet with your marketing team
+
+-  **Go through the list of scripts and give your view on each script**.
+   For instance, are any of the nice-to-have scripts that are very damaging for site
```
Comment (kaycebasques) [inline src/site/content/en/blog/fixing-website-speed-cross-functionally/index.md]:
```suggestion
   For instance, are any of the nice-to-have scripts very damaging to site
```

**Comment by kaycebasques** [comment]
@MichaelSolati I was originally going to post this under web.dev/blog and then realized that it would be better under web.dev/fast. I can't tell why the build is barfing. Can you?

**Comment by MichaelSolati** [comment]
@kaycebasques sorry for taking long, but it turned out to be a simple fix, just had to add `layout: post` to the head of the markdown. I pushed up the fix.

File: src/site/content/en/fast/fixing-website-speed-cross-functionally/index.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
+---
+title: Fixing website speed cross-functionally
+subhead: How other departments can help make your website speed optimization project a bigger success.
+authors:
+  - lchansson
+description: How other departments can help make your website speed optimization project a bigger success.
+date: 2020-02-28
+tags:
+  - speed
```
[the remark this answers is not in the document]
Comment (kaycebasques) [reply src/site/content/en/fast/fixing-website-speed-cross-functionally/index.md]:
```suggestion
  - performance
```

**Pull request merged**